search-app/src/cmr/search/api/tags.clj

```diff
@@ -23,16 +24,38 @@
   [data]
   (some #(contains? % :errors) data))
 
+(defn tag-association-results->status-code
+  "Check for concept-types requiring error status to be returned.
+  If the concept-type is error-sensitive the function will check for any errors in the results.
+  Will return:
+  - 200 OK -- if response has no errors
+  - 207 MULTI-STATUS -- if response has some errors and some successes
+  - 400 BAD REQUEST -- if response has all errors"
+  [results]
+    (let [result-count (count results)
+          num-errors (assoc/num-errors-in-assoc-results results)]
+      (if (= 0 result-count)
+        200
+        (if (= num-errors result-count)
+          400
+          (if (> num-errors 0)
+            207
+            200)))))
+
 (defn tag-api-response
   "Creates a successful tag response with the given data response"
   ([data]
    (if (has-error? data)
      (tag-api-response 400 data)
      (tag-api-response 200 data)))
   ([status-code data]
-   {:status status-code
-    :body (json/generate-string (util/snake-case-data data))
-    :headers {"Content-Type" mt/json}}))
+   (if (= 207 status-code)
+     {:status status-code
+      :body (json/generate-string (util/snake-case-data (cmr.search.api.association/add-individual-statuses data)))
```

Comment:
cmr.search.api.association, you simplify by adding path to imports?

Reply:
oh my how did that get in there

Reply:
fixed